games/tic_tac_toe.c

```diff
@@ -80,6 +82,27 @@ int main()
  *
  * @returns None
  */
+int TakeInput(){
+    
+    // Input player's choice of placing 'x' or 'o' in variable x 
+    char x[100];
+    fgets(x, 100, stdin);
+
+    // to check the length of input!
+    int i;
+    for (i = 0; x[i] != '\0'; i++);   
```

Comment:
in order to ensure this is valid, should the memory block of `x` be initialized to `\0` as:
```c
memset(x, 100, '\0');
```